packages/angular/ssr/src/routes/route-config.ts

```diff
@@ -158,6 +162,16 @@ export interface ServerRoutePrerenderWithParams extends Omit<ServerRoutePrerende
    *       return ids.map(id => ({ id })); // Generates paths like: [{ id: '1' }, { id: '2' }, { id: '3' }]
    *     },
    *   },
+   *   {
+   *     path: '/product/:id/**',
+   *     renderMode: RenderMode.Prerender,
+   *     async getPrerenderParams() {
+   *       return [
+   *         { id: '1', '**': '/laptop/123' },
+   *         { id: '2', '**': '/laptop/456' }
```

Comment:
Nit: Is the leading slash necessary? If not, I'd probably leave it off given that this is replacing a `**` which is already preceded by a slash.

Reply:
Not it is not